src/backends/js-testing.js

```diff
@@ -112,10 +115,15 @@ export default class JSTestingBackend extends BaseBackend {
     }
 
     async takeScreenshot (id, screenshotPath) {
-        var buffer = await requestApi(BROWSERSTACK_API_PATHS.screenshot(this.workers[id].id));
-        var image  = await jimp.read(buffer);
+        var buffer      = await requestApi(BROWSERSTACK_API_PATHS.screenshot(this.workers[id].id));
```

Comment:
This is a [`jpeg` with `png`extention](https://github.com/browserstack/api/issues/33), so this fix would unfortunately not work.

Reply:
Thank you for your remark. I changed using `pngjs` to `sharp`.